CHANGELOG.md

```diff
@@ -126,6 +126,7 @@ The present file will list all changes made to the project; according to the
   We added a compatibility layer to handle main usages found in plugins, but we cannot ensure compatibility with all properties and methods that were inherited from `PHPMailer\PHPMailer\PHPMailer`.
 - `CommonGLPI::$othertabs` static property has been made private.
 - `CommonGLPI::createTabEntry()` signature changed.
+- `CommonITILValidation::showSummary()` method has been made private.
```

Comment:
Changelog should probably include something about the new feature from a user POV in addition to the code API changes.

Reply:
good catch. I suggest to add in `Added` section : 
```
- Approval requests are grouped by validation step.
- Validation step threshold can be changed for each ticket
- Validation step can be assigned by rule (and threshold can be changed)
 ```
Is it ok for you ?

Reply:
I think @cedric-anne has made to the API, so he can add a line. For my part, I'll add '- Approval requests are grouped by validation step. '